packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -573,6 +625,14 @@ export function TransferPanelMain({
 
   useUpdateUSDCTokenData()
 
+  type NetworkListboxProps = {
+    disabled?: boolean
+    label: string
+    options: Chain[]
+    value: Chain
+    onChange: (value: Chain) => void
+  }
+
   type NetworkListboxesProps = {
     from: Pick<NetworkListboxProps, 'onChange'>
     to: Omit<NetworkListboxProps, 'label'>
```

Comment:
i would just do and delete the other props from `to` in the return object and `getDestinationChains` because they are called within `NetworkSelectionContainer` already
```suggestion
    from: {
      onChange: (value: Chain) => void
    }
    to: {
      onChange: (value: Chain) => void
    }
```